server/mvc/log.js

```diff
@@ -0,0 +1,4 @@
+module.exports.init = function(log){
```

Comment:
I guess this will have more stuff inside, as I don't really see the point here :octopus: